### PR TITLE
fix: admin management api minor refactoring

### PR DESCRIPTION
**Pull request by ianliuwk1019**
- Rename some tests filenames.
- Add comments for schema.py for convention.
- Add suffix for some schema objects.
- Update admin management api openapi.json.
- Recreate admin management api client code.
- Configure frontend for FAMAccessControlPrivilegeApi
- Fix frontend issue due to backend schema change.
- Use service instance dependency in routers.

### CODE REVIEW

File: server/admin_management/api/app/schemas.py [hunk 1]
```diff
@@ -57,60 +65,60 @@ class FamUserBase(BaseModel):
...
 
-class FamUserCreate(FamUserBase):
+class FamUserDto(FamUserBase):
     create_user: Annotated[str, StringConstraints(max_length=60)]
 
     model_config = ConfigDict(from_attributes=True)
 
 
-class FamUserType(BaseModel):
+class FamUserTypeDto(BaseModel):
```
Comment (MCatherine1994) [inline server/admin_management/api/app/schemas.py:74]: Hi Ian, I wonder if "DTO" schema is when we use to create the record in db? or it's just a naming convention, if it's not used at router level, then we can call it with "Dto"?

**Comment by MCatherine1994** [comment]
thanks Ian! I just got a question, looks good!

**Review by MCatherine1994 — APPROVED** [review]

File: server/admin_management/api/app/schemas.py (hunk 1 above)
Comment (ianliuwk1019) [reply server/admin_management/api/app/schemas.py:74]: Hi @MCatherine1994 , in layered pattern, "DTO" object is a "transfer" object. It is **not specific** to repository insert/update/select. It only means a **transfer object between layers** (presentation layer, service layer, repository layer etc...). Normally you would see DTO lives in service layer **a lot** but having conversion from presentation layer and repository layer. Layered pattern uses it to abstract away business/responsibility/dependency between each layer.
Although I stick to convention using "DTO", but in python, our code, we did not have strict disinclination for layers yet (and probably does not need to go that far). If you prefer, we can drop the "DTO" and just keep the (__request and __response) if they are really being used at endpoints.

Comment (MCatherine1994) [reply server/admin_management/api/app/schemas.py:74]: I'm fine with it, thanks Ian!!

**Pull request merged**